webapp-django/crashstats/settings/__init__.py

```diff
@@ -1,8 +1,6 @@
 from .base import *
 try:
     from .local import *
-except ImportError, exc:
-    exc.args = tuple(
-        ['%s (did you rename settings/local.py-dist?)' % exc.args[0]]
-    )
-    raise exc
+except ImportError:
+    print ('WARNING - no local.py'
+           ' (deprecated, use environment variables instead)')
```

Comment:
@peterbe sorry I don't understand:

> Can you move this to just after ```from .local import *```

Isn't it already just after it?

Also - I changed from ```print``` to ```warnings.warn``` but I don't see it printed when I do e.g. ```./manage.py runserver```, is that because logging is disabled by default?